### PR TITLE
[FlexibleHeader] Add WKWebView considerations docs.

### PR DESCRIPTION
**Pull request by jverkoey**
This is a follow-up to 4e658a5aee98b6d9664d6e890e40de5f68125494.

### CODE REVIEW

File: components/FlexibleHeader/README.md [hunk 1]
```diff
@@ -805,6 +806,89 @@ the flexible header staying fixed in place, even though the underlying scroll vi
...
+<!--</div>-->
+
+If you support any OS below iOS 11, you'll **also** need to adjust the frame of your WKWebView on
+devices running these older operating systems so that the web view is aligned to the top layout
+guide.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+// No swift example available. See the Objective-C example instead.
```
Comment (romoore) [inline components/FlexibleHeader/README.md]: Why is swift not available? Is our fix incompatible with Swift?

**Review by romoore — DISMISSED** [review]
Concerned that our code won't work with Swift.

File: components/FlexibleHeader/README.md (hunk 1 above)
Comment (jverkoey) [reply components/FlexibleHeader/README.md]: Fixed.

Comment (romoore) [reply components/FlexibleHeader/README.md]: Maybe you forgot to push the change?

Comment (jverkoey) [reply components/FlexibleHeader/README.md]: Hadn't written it yet :)

**Comment by ianegordon** [comment]
LGTM.  Kokoro may be having problems deleting a label.

**Comment by jverkoey** [comment]
Waiting on LGTM for this PR.

File: components/FlexibleHeader/README.md (hunk 1 above)
Comment (romoore) [reply components/FlexibleHeader/README.md]: Is this still missing?

Comment (jverkoey) [reply components/FlexibleHeader/README.md]: Nope it's here now.

Comment (jverkoey) [reply components/FlexibleHeader/README.md]: Augh. Pushed the branch >.<

**Review by randallli — APPROVED** [review]

**Pull request merged**